src/parakeet/pose.py

```diff
@@ -0,0 +1,50 @@
+from dataclasses import dataclass
+
+import numpy as np
+from scipy.spatial.transform import Rotation as R
+
+
+@dataclass
+class PoseSet:
+    """A description of poses of the system being simulated.
+
+    Parameters
+    __________
+
+    orientations : np.ndarray
+        (n, 3, 3) array of rotation matrices describing the pose of the system.
+    shifts : np.ndarray
+        (n, 3) array of how to shift the sample volume (units: A)
+    """
+    orientations: np.ndarray
+    shifts: np.ndarray
+
+    def __post_init__(self):
+        self._orientations = self.orientations
+        self.orientations = R.from_matrix(self._orientations)
```

Comment:
The dataclass decorator is new to me but as I understand it, the purpose is to remove the need for defining __init__ functions manually in certain cases. In this case since we need to add a __post_init__ function perhaps it would be easier to define the class in the usual way.

Reply:
yeah whichever you prefer, happy to change!